packages/android_intent/android/src/main/java/dev/flutter/plugins/androidintent/MethodCallHandlerImpl.java

```diff
@@ -0,0 +1,132 @@
+package dev.flutter.plugins.androidintent;
+
+import android.content.ComponentName;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.Settings;
+import android.text.TextUtils;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
+import io.flutter.plugin.common.MethodChannel.Result;
+import java.util.ArrayList;
+import java.util.Map;
+
+/** Forwards incoming {@link MethodCall}s to {@link IntentSender#send}. */
+public class MethodCallHandlerImpl implements MethodCallHandler {
```

Comment:
Given that this is public, would it make sense to specify what kind of method calls this is handling? e.g., `IntentMethodCallHandlerImpl`?

Reply:
I think that's already implied from the package name, but I don't feel super strongly on this if you'd still prefer a rename.

Reply:
Ok. You may want to unify the approach on the team. The connectivity plugin chose to prefix:
https://github.com/flutter/plugins/pull/2142

Either seems acceptable, but it also seems like one of those details that doesn't have any cause to randomly change between 1P plugins.